### PR TITLE
Update notes for update existing pack

### PR DESCRIPTION
**Pull request by barchen1**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/29243

## Description
Added notes about adding new content entities to existing packs.

## Screenshots
Screen Shot 2021-11-25 at 10 43 31
Screen Shot 2021-11-25 at 10 44 02

### CODE REVIEW

**Review by ShahafBenYakir — CHANGES_REQUESTED** [review]
1. Both 2 and 3 are pretty similar I would unify them like this:
    ```suggestion
    2. The contribution mode selection dropdown will allow two types of changes:
       - Suggested packs - Packs suggested by the system (We check if your content item was duplicated from another item).
       - All packs - A list of all existing packs you can contribute to.
    ```
2. What does the last limitation say? it sounds like we are saying that we are not supporting resubmission of packs, tho two paragraphs above it we say we do support.

**Comment by barchen1** [comment]
@ShahafBenYakir I think it would be better to keep both 2 and 3- cause the explanation regarding the duplicated content in the second paragraph is important.
The last limitation says that we are not supporting resubmission to existing packs (https://github.com/demisto/etc/issues/43197) (we do support resubmission to new packs).

**Comment by ShahafBenYakir** [comment]
1. It is too technical and confusing, a simple contributor will not understand what we want here. What is the important part of item 3 (the underline)?
2. It is not very clear, let's talk offline.

File: docs/contributing/marketplace.md
Comment (avidan-H) [inline docs/contributing/marketplace.md:36]:
```suggestion
	_Contribute an update an existing content entity_: Duplicate the entity you want to update. For example, if you want to update the `AbuseIPDB` integration, you need to duplicate it and save the duplicate with your changes as `AbuseIPDB_copy`.
```

File: docs/contributing/marketplace.md
Comment (avidan-H) [inline docs/contributing/marketplace.md:39]:
```suggestion
4. After you submit the form you will be redirected to a page that informs you that your contribution was received and is being processed.  
```

File: docs/contributing/marketplace.md
Comment (avidan-H) [inline docs/contributing/marketplace.md:40]: For number 6 I think you can remove this bit,
`(Each contributor can only modify files in content packs that they contributed).`

For number 7, I think language-wise it can be rewritten to be more accurate (some of this may no longer be true/accurate so verify that this is actually the behavior still today),
```
    A documentation template is automatically generated for the content entities included in your contribution.
    You will need to review the documentation (README.md) and modify it according to XSOAR standards.
    The files to be reviewed will be listed in a comment on the pull request.
    See the [Documentation article](https://xsoar.pan.dev/docs/documentation/readme_file) for more information.
```

File: docs/contributing/marketplace.md
Comment (avidan-H) [inline docs/contributing/marketplace.md:60]:
```suggestion
      1. Create or edit any content items that need to be included in your contribution.
```

File: docs/contributing/marketplace.md
Comment (avidan-H) [inline docs/contributing/marketplace.md:72]:
```suggestion
      5. Once the changes are pushed to your branch, you will receive a notification via email.
```

File: docs/contributing/marketplace.md
Comment (avidan-H) [inline docs/contributing/marketplace.md:75]: I think this can be removed.

**Review by avidan-H — CHANGES_REQUESTED** [review]

**Comment by barchen1** [comment]
This PR is waiting for https://github.com/demisto/contribution-management/pull/160 to be merged.

**Comment by barchen1** [comment]
Will merge this PR once we merged the fix in https://github.com/demisto/contribution-management/pull/160

**Pull request closed without merging**
